receiver/awsecscontainermetricsreceiver/factory.go

```diff
@@ -16,29 +16,37 @@ package awsecscontainermetricsreceiver
 
 import (
 	"context"
+	"fmt"
+	"os"
 	"time"
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config/configmodels"
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/receiver/receiverhelper"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecscontainermetricsreceiver/awsecscontainermetrics"
 )
 
+// Factory for awscontainermetrics
 const (
+	// Key to invoke this receiver (awsecscontainermetrics)
 	typeStr = "awsecscontainermetrics"
 
-	// Default collection interval
+	// Default collection interval. In every 20s the receiver will collect metrics from Amazon ECS Task Metadata Endpoint
```

Comment:
```suggestion
	// Default collection interval. Every 20s the receiver will collect metrics from Amazon ECS Task Metadata Endpoint
```

Reply:
Is it task metadata endpoint or container metrics endpoint?

Reply:
Amazon ECS Task Metadata [Endpoint](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint.html)